.gitignore

```diff
@@ -142,6 +142,8 @@ xdem/version.py
 # Example data downloaded/produced during tests
 examples/data/Longyearbyen/data/
 examples/data/Longyearbyen/processed/
+examples/data/*.tif
```

Comment:
I think more importantly, we should try to clean our test data after the tests...

Reply:
Yes, deleting the files could be run as a final test, when all tests pass. Maybe we can make sure the files are not deleted until the tests pass, so it would not be a problem re-running individual files?

Reply:
OK, I put this as an issue